test/cases/fedora_30-aarch64-ami-boot.json

```diff
@@ -8,7 +8,7 @@
     "image-type": "ami",
     "repositories": [
       {
-        "baseurl": "http://dl.fedoraproject.org/pub/fedora/linux/releases/30/Everything/aarch64/os/",
+        "baseurl": "http://fedora.mirrors.tds.net/fedora/releases/30/Everything/aarch64/os/",
```

Comment:
If you replace with `download.fedoraproject.org` it should automatically point to the closest mirror, no ?
 I prefer not hard-coding a specific local mirror in the sources.

Reply:
@atodorov This was mainly a hack just to see if the job run time improves. I don't want to put it into production like this. :smile: